tests/e2e/default/contacts/edit.wdio-spec.js

```diff
@@ -40,6 +40,7 @@ describe('Edit contacts with the default config. ', () => {
   });
 
   it('should remove the primary contact from the clinic when the contact is deleted', async () => {
+    await commonPage.refresh();
```

Comment:
Is this to fix this issue? https://github.com/medic/cht-core/issues/8097

Consider a separate PR?